### PR TITLE
chore(doc): update benchmarks timings

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

**Comment by IceTDrinker** [comment]
cargo build is red

File: tfhe/docs/getting_started/benchmarks.md [hunk 1]
```diff
@@ -29,6 +29,20 @@ The next table shows the operation timings on CPU when all inputs are encrypted:
...
+| Comparisons  (`ge`, `gt`, `le`, `lt`)                  |  57.1 ms    | 72.9 ms     | 93.0 ms     | 116 ms      | 138 ms       | 164 ms       |
+| Max / Min   (`max`,`min`)                              |  94.3 ms    | 114 ms      | 138 ms      | 159 ms      | 189 ms       | 233 ms       |
+| Bitwise operations (`&`, `\|`, `^`)                    |  19.6 ms    | 20.1 ms     | 20.2 ms     | 21.7 ms     | 23.9 ms      | 25.7 ms      |
+| Div / Rem  (`/`, `%`)                                  |  711 ms     | 1.81 s      | 4.43 s      | 10.5 s      | 25.1 s       | 63.2 s       |
+| Left / Right Shifts (`<<`, `>>`)                       |  99.5 ms    | 125 ms      | 155 ms      | 190 ms      | 234 ms       | 434 ms       |
+| Left / Right Rotations (`left_rotate`, `right_rotate`) |  101 ms     | 125 ms      | 154 ms      | 188 ms      | 234 ms       | 430 ms       |
+| Leading / Trailing zeros/ones                          |  96.7 ms    | 155 ms      | 181 ms      | 241 ms      | 307 ms       | 367 ms       |
+| Log2                                                   |  112 ms     | 176 ms      | 200 ms      | 265 ms      | 320 ms       | 379 ms       |
```
Comment (IceTDrinker) [inline tfhe/docs/getting_started/benchmarks.md:45]: we need to replace the old array right ?

File: tfhe/docs/getting_started/benchmarks.md [hunk 2]
```diff
@@ -101,9 +128,9 @@ scripts/build-openfhe-development-hexl.sh
 Using the same hpc7a.96xlarge machine as the one for tfhe-rs, the timings are as follows:
 
 | Parameter set                     | GINX    | GINX w/ Intel HEXL |
-| --------------------------------- | ------- | ------------------ |
-| FHEW\_BINGATE/STD128\_OR          | 25.5 ms | 21,6 ms            |
-| FHEW\_BINGATE/STD128\_LMKCDEY\_OR | 25.4 ms | 19.9 ms            |
+| --------------------------------- | ------- |--------------------|
+| FHEW\_BINGATE/STD128\_OR          | 25.5 ms | 24,0 ms            |
+| FHEW\_BINGATE/STD128\_LMKCDEY\_OR | 25.4 ms | 23.6 ms            |
```
Comment (IceTDrinker) [inline tfhe/docs/getting_started/benchmarks.md:133]: they are slower ?

**Comment by IceTDrinker** [comment]
adding @agnesLeroy to check GPU timings

File: tfhe/docs/getting_started/benchmarks.md [hunk 3]
```diff
@@ -46,6 +60,19 @@ The next table shows the operation timings on CPU when the left input is encrypt
...
+| Equal / Not Equal (`eq`, `ne`)                         | 31.2 ms    | 30.9 ms     | 34.4 ms     | 54.5 ms     | 57.0 ms      | 58.0 ms      |
+| Comparisons  (`ge`, `gt`, `le`, `lt`)                  | 38.6 ms    | 56.3 ms     | 76.1 ms     | 99.0 ms     | 124 ms       | 141 ms       |
+| Max / Min   (`max`,`min`)                              | 74.0 ms    | 103 ms      | 122 ms      | 144 ms      | 171 ms       | 214 ms       |
+| Bitwise operations (`&`, `\|`, `^`)                    | 19.0 ms    | 19.8 ms     | 20.5 ms     | 21.6 ms     | 23.8 ms      | 25.8 ms      |
+| Div  (`/`)                                             | 192 ms     | 255 ms      | 322 ms      | 459 ms      | 877 ms       | 2.61 s       |
+| Rem  (`%`)                                             | 336 ms     | 482 ms      | 650 ms      | 871 ms      | 1.39 s       | 3.05 s       |
+| Left / Right Shifts (`<<`, `>>`)                       | 19.5 ms    | 20.2 ms     | 20.7 ms     | 22.1 ms     | 23.8 ms      | 25.6 ms      |
+| Left / Right Rotations (`left_rotate`, `right_rotate`) | 19.0 ms    | 20.0 ms     | 20.8 ms     | 21.7 ms     | 23.9 ms      | 25.7 ms      |
+
```
Comment (IceTDrinker) [inline tfhe/docs/getting_started/benchmarks.md:75]: same here

File: tfhe/docs/getting_started/benchmarks.md (hunk 2 above)
Comment (soonum) [reply tfhe/docs/getting_started/benchmarks.md:133]: Yes it seems so

**Pull request merged**